test/unit/specs/components/ActionModal/components/TableInvoice.spec.js

```diff
@@ -6,25 +6,31 @@ describe(`TableInvoice`, () => {
   const localVue = createLocalVue()
   localVue.directive(`tooltip`, () => {})
 
-  const $store = {
-    getters: {
-      bondDenom: `STAKE`
-    }
-  }
-
   beforeEach(() => {
     wrapper = shallowMount(TableInvoice, {
       localVue,
-      mocks: { $store },
       propsData: {
-        amount: 1,
+        amount: 17.2,
         gasEstimate: 1234567,
```

Comment:
can we get rid of gasEstimate and gasPrice right away? Doesn't make sense to keep this right?

Reply:
Can't get rid of gasEstimate because it is a required prop in TableInvoice but gasPrice we can.